.github/workflows/golangci-lint.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches:
       - 'main'
       - '[0-9]+.[1-9][0-9]*.x'
+    paths-ignore:
```

Comment:
i might extract this into an own pull request

Reply:
@mowies what do you think? should i open a issue/pr for this?

Reply:
i need some more context please

Reply:
sorry the comment is badly based from myside it is about adding 

```
paths-ignore:
  - 'docs/**'
```

for all go build relevant builds, to reduce the usage of github actions, and also to give contributors a faster feedback loop. i do not need to golint, unittests etc when writing documentation

Reply:
ok cool, yeah then let's move this into a separate issue :)

Reply:
i will create a follow up as discussed within the community meeting today